src/channel.cpp

```diff
@@ -725,6 +728,13 @@ void CChannel::PrepAndSendPacket ( CHighPrioSocket*        pSocket,
                                    const CVector<uint8_t>& vecbyNPacket,
                                    const int               iNPacketLen )
 {
+    // From v3.8.0 onwards, a server will not send audio to a client until that client has sent channel info.
+    // This addresses #1243 but means that clients earlier than v3.3.0 (24 Feb 2013) will no longer be compatible.
+    if ( bIsServer && !bIsIdentified )
```

Comment:
Why not send silence?

Reply:
I don't see any point in doing so. And it's only for a very short period until the client sends its info.

Reply:
The current code in this PR also has the slight side effect of increasing the required effort for amplification attacks a tiny bit (more understanding, more bytes). Sending silence would negate that and would require more code. I'd be happier with the send-nothing approach.

Reply:
OK, good reason.  And if it's been tried out and seems to work, fair enough -- I was just being cautious about changes in behaviour that the client isn't expecting.